source/test/main.test.ts

```diff
@@ -71,4 +71,22 @@ test("has output - bucketName", () => {
     const stack = new MainStack(app, 'test');
     const template = Template.fromStack(stack);
     template.hasOutput('bucketName', {})
-})
\ No newline at end of file
+})
+
+test("has 1 CustomResource", () => {
+    const app = new App();
+    const stack = new MainStack(app, 'test');
+    const template = Template.fromStack(stack);
+    template.hasResource('AWS::CloudFormation::CustomResource', 1)
+})
+
+test("The CustomResource has Condition", () => {
```

Comment:
you should make sure your condition is expected(the region is `us-west-2`)

Reply:
add region check